integration/has_vm/has_vm_test.go

```diff
@@ -125,7 +125,7 @@ var _ = Describe("BOSH Director Level Integration for has_vm", func() {
 
 			outputBytes, err := testhelperscpi.RunCpi(rootTemplatePath, tmpConfigPath, jsonPayload)
 			Expect(err).ToNot(HaveOccurred())
-			
+
```

Comment:
For some reasons it seems that this file keeps getting new lines from your commits... Can we fix that? No new line needed between the call and check for error. That's the style we have used consistently and this is the only visible/known culprit. I am guessing it's a `go vet` Issue or specific to your env.